test-case/src/main/java/pkg/UsingMarker.java

```diff
@@ -13,8 +14,10 @@
   private final Logger logger = LoggerFactory.getLogger(getClass());
 
   void method() {
-    Marker marker = null;
+    Marker marker = MarkerFactory.getMarker("my marker");
     logger.error(marker, "Hello, marker");
+    logger.error(marker, "Hello, {}");
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Not enough arguments. ([squid:S2275](https://sonarcloud.io/organizations/kengotoda-github/rules#rule_key=squid%3AS2275))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=jp.skypencil.findbugs.slf4j%3Afindbugs-slf4j&pullRequest=105&issues=AWZbKQND9tx2us2yCfhh&open=AWZbKQND9tx2us2yCfhh)

[ISSUE_KEY]: AWZbKQND9tx2us2yCfhh